source/system/external/stdatomic.ooc

```diff
@@ -0,0 +1,27 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+include ./atomic
+
+atomic_int: extern cover
```

Comment:
This is probably the most idiomatic way to write bridges to C, since most libraries are written in a pseudo-OO way.  

Usually you do a cover from the pointer type and just define the extern functions as methods but in this case it makes more sense to have the cover be by value.  

If anyone is interested in some more advanced idiomatic ways to write bridges, I would recommend [oocui](https://github.com/shamanas/oocui), the IUP bindings, which include simple covers from pointer types as well passing Closures and C varargs.